web-www/pages/index.tsx

```diff
@@ -0,0 +1,137 @@
+  EcologicalCreditCardsSectionDocument,
+  EcologicalCreditCardsSectionQuery,
+  HomeFoldSectionDocument,
+  HomeFoldSectionQuery,
+  LedgerSectionDocument,
+  LedgerSectionQuery,
+  MarketplaceSectionDocument,
+  MarketplaceSectionQuery,
+  PartnersSectionDocument,
+  PartnersSectionQuery,
+  StatsSectionDocument,
+  StatsSectionQuery,
+  ValuesSectionDocument,
+  ValuesSectionQuery,
+} from '@/generated/sanity-graphql';
+import { sanityClient } from '@/lib/clients/sanityClient';
+
+export default function Home({
+  homeFoldData,
+  carbonPlusData,
+  climateData,
+  marketplaceData,
+  partnersData,
+  ecologicalCreditCardsData,
+  statsData,
+  valuesData,
+  ledgerData,
+  blogData,
+}: InferGetStaticPropsType<typeof getStaticProps>) {
+  return (
+    <>
+      <Head>
+        <title>Create Next App</title>
+        <meta name="description" content="Generated by create next app" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <Box component="main" sx={{ overflow: 'hidden' }}>
+        <HomeFoldSection homeFoldData={homeFoldData} />
+        <CarbonplusSection carbonPlusData={carbonPlusData} />
+        <ClimateSection climateData={climateData} />
+        <MarketplaceSection marketplaceData={marketplaceData} />
+        <PartnersSection partnersData={partnersData} />
+        <EcologicalCreditCardsSection
+          ecologicalCreditCardsData={ecologicalCreditCardsData}
+        />
+        <StatsSection statsData={statsData} />
+        <HomeValues valuesData={valuesData} />
+        <HomeLedger ledgerData={ledgerData} />
+        <BlogSection blogData={blogData} />
+      </Box>
+    </>
+  );
+}
+
+export const getStaticProps = async () => {
+  const [
+    homeFoldData,
+    carbonPlusData,
+    climateData,
+    marketplaceData,
+    partnersData,
+    ecologicalCreditCardsData,
+    statsData,
+    valuesData,
+    ledgerData,
+    blogData,
+  ] = await Promise.all([
+    sanityClient.query<HomeFoldSectionQuery>({
+      query: HomeFoldSectionDocument,
+    }),
+    sanityClient.query<CarbonPlusSectionQuery>({
+      query: CarbonPlusSectionDocument,
+    }),
+    sanityClient.query<ClimateSectionQuery>({
+      query: ClimateSectionDocument,
+    }),
+    sanityClient.query<MarketplaceSectionQuery>({
+      query: MarketplaceSectionDocument,
+    }),
+    sanityClient.query<PartnersSectionQuery>({
+      query: PartnersSectionDocument,
+    }),
+    sanityClient.query<EcologicalCreditCardsSectionQuery>({
+      query: EcologicalCreditCardsSectionDocument,
+    }),
+    sanityClient.query<StatsSectionQuery>({
+      query: StatsSectionDocument,
+    }),
+    sanityClient.query<ValuesSectionQuery>({
+      query: ValuesSectionDocument,
+    }),
+    sanityClient.query<LedgerSectionQuery>({
+      query: LedgerSectionDocument,
+    }),
+    sanityClient.query<BlogSectionQuery>({
+      query: BlogSectionDocument,
+    }),
```

Comment:
In order to save some time for the POC, the same fetching queries from the current site have been reused.
We could improve this part by having a single query.

Reply:
Actually, I already fix it, will be cleaner: https://github.com/regen-network/regen-web/pull/1867/commits/f5b37e8895ec8ca64d98c42e724648d549f735c7